trace-sdk/src/main/java/io/bitrise/trace/data/management/formatter/cpu/SystemCpuDataFormatter.java

```diff
@@ -47,9 +47,6 @@
         }
 
         final CpuStat systemCpuStat = (CpuStat) data.getContent();
-        if (systemCpuStat == null) {
```

Comment:
likewise, L45 covers this scenario - if it's a null value, it's not a CpuStat and so it gets caught there.